### DISCUSSION TITLE
sending mail: failed to get recipient: 550 requested action not taken: mailbox unavailable or not local

### DISCUSSION THREAD

**Discussion by mondster2602** [Security-Onion-Solutions/securityonion#1946] [Unsupported Versions] [answered]
Having trouble setting up smtp in kolide fleet. Having error (see title)

Can someone help me with this?

Im using my company's SMTP
![16056072731517712174371738459955](https://user-images.githubusercontent.com/74595279/99375660-f3adf400-28fe-11eb-9757-e60fac92d00b.jpg)

**Comment by weslambert** [comment, marked as answer]
I'd have to look further, but I'd surmise that the appropriate SMTP config/relay is not set up inside of the Fleet container.  Given that we are moving to FleetDM, it may be best to wait until then to look at this.

**Reply by mondster2602** [reply]
Thanks. Keep me posted please.
